sa/sa.go

```diff
@@ -64,6 +64,18 @@ type authzModel struct {
 	core.Authorization
 }
 
+// OrderFQDNSet contains the SHA256 hash of the lowercased, comma joined names
```

Comment:
Fix capitalization to match new unexported name

Reply:
Fixed in aaaef42